src/gurt/telemetry.c

```diff
@@ -1075,6 +1075,49 @@ d_tm_compute_histogram(struct d_tm_node_t *node, uint64_t value)
 	return rc;
 }
 
+/**
+ * Set the given counter to the specified \a value
+ *
+ * \param[in,out]	metric	Pointer to the metric
+ * \param[in]		value	Increments the counter by this \a value
```

Comment:
Should be: "Sets the counter to this value"

Reply:
fixed.